timemachine/cpp/src/neighborlist.hpp

```diff
@@ -55,15 +55,15 @@ template <typename RealType> class Neighborlist {
     void compute_block_bounds_host(
         const int N, const double *h_coords, const double *h_box, double *h_bb_ctrs, double *h_bb_exts);
 
-    unsigned int *get_ixn_atoms() { return d_ixn_atoms_; }
+    unsigned int *get_ixn_atoms() { return d_ixn_atoms_; };
```

Comment:
were these extra semi colons required? (not sure they're needed if we're defining the function, as opposed to just declaring them)

Reply:
I had run into some compilation issues that seemed like they were due to missing semi-colons and I noticed these trying to find the issue. I left them to be consistent with our other code, but not actually of any importance.